src/one_time/core.clj

```diff
@@ -4,6 +4,15 @@
             [one-time.totp  :as totp]
             [one-time.hotp  :as hotp]))
 
+(defn ^:private ct=
+  "Constant-time comparison for numeric types, via bytewise string equality"
+  [x y]
+  (let [x' (.getBytes (str x))
+        y' (.getBytes (str y))]
+    (and
+     (= (count x') (count y'))
+     (->> (map bit-xor x' y') (reduce +) (= 0)))))
```

Comment:
I'd prefer if the intent of this expression was a bit easier to follow. Can we use this instead ?
https://github.com/weavejester/crypto-equality/blob/master/src/crypto/equality.clj#L11